dm/worker/upgrade.go

```diff
@@ -96,7 +99,7 @@ func loadVersion(h dbOperator) (ver version, err error) {
 	data, err := h.Get(dmWorkerVersionKey, nil)
 	if err != nil {
 		if err == leveldb.ErrNotFound {
-			log.Warnf("[worker upgrade] no version found in levelDB, default %s used", defaultPreviousWorkerVersion)
+			log.L().Warn("no version found in levelDB, use default version", zap.String("feature", "bootstrap upgrade"), zap.Stringer("default version", defaultPreviousWorkerVersion))
```

Comment:
we can pre-construct the log field used in the whole file, like this `zap.String("feature", "bootstrap upgrade")`?

Reply:
I suggest to do it when we have bootstrap component, I have dealt with it briefly at the moment